### PR TITLE
[1LP][RFR] Tests for grouping candu graphs by tags

### PR DESCRIPTION
**Pull request by nachandr**
I have added tests for grouping C&U graphs by tags.

PRT results:
510 - PASSED

### CODE REVIEW

File: cfme/fixtures/candu.py [hunk 1]
```diff
@@ -64,3 +68,37 @@ def candu_tag_vm(provider, enable_candu_category):
...
+    app.evmserverd.start()
+    return app
+
+
+@pytest.fixture(scope="module")
+def candu_db_restore(temp_appliance_extended_db):
+    app = temp_appliance_extended_db
+    # get DB backup file
+    db_storage_hostname = conf.cfme_data.bottlenecks.hostname
+    db_storage_ssh = SSHClient(hostname=db_storage_hostname, **conf.credentials.bottlenecks)
```
Comment (digitronik) [inline cfme/fixtures/candu.py]: :+1:

File: cfme/tests/candu/test_graph_groupbytag.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        * Zoom graph to get Table
+        * Check tag assigned to VM available in chart legends
+        * Compare table and graph data
+
+    Bugzillas:
+        * 1367560
+
+    Polarion:
+        assignee: nachandr
+        initialEstimate: 1/4h
```
Comment (digitronik) [inline cfme/tests/candu/test_graph_groupbytag.py]: @nachandr can you please add `casecomponent`? also without `Tire` we unable to decide `caselevel`. so please add docstring changes.

File: cfme/tests/candu/test_graph_groupbytag.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    """
+    view = navigate_to(host, 'candu')
+    back_date = datetime.now() - timedelta(days=1)
+    data = {'interval': interval, 'group_by': gp_by}
+
+    # Note: If we won't choose backdate then  we have wait for 30min at least for metric collection
```
Comment (digitronik) [inline cfme/tests/candu/test_graph_groupbytag.py]: I think with db collection it will not required.

Comment (nachandr) [reply cfme/tests/candu/test_graph_groupbytag.py]: This is still needed, I have added a comment as to why it's needed in the test module.

File: cfme/tests/candu/test_graph_groupbytag.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    # pre-existing C&U data.
+    if interval == "Hourly":
+        data.update({"calendar": back_date})
+    view.options.fill(data)
+
+    # Check graph displayed or not
+    try:
+        graph = getattr(view.interval_type, graph_type)
+    except AttributeError as e:
+        logger.error(e)
```
Comment (izapolsk) [inline cfme/tests/candu/test_graph_groupbytag.py]: @ganeshhubale, please explain why just logging w/o exception or fail

Comment (nachandr) [reply cfme/tests/candu/test_graph_groupbytag.py]: @izapolsk , I'm now raising an exception.

File: cfme/tests/candu/test_graph_groupbytag.py [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    """
+    view = navigate_to(host, 'candu')
+    data = {'interval': interval, 'group_by': gp_by}
+    view.options.fill(data)
+
+    # Check graph displayed or not
+    try:
+        graph = getattr(view.interval_type, graph_type)
+    except AttributeError:
+        raise Exception('Graph {} not found'.format(graph_type))
```
Comment (izapolsk) [inline cfme/tests/candu/test_graph_groupbytag.py]: I'd rather call `pytest.fail('bla bla)`.
Test should rather fail in this case than become broken

**Comment by izapolsk** [comment]
@nachandr, please fix above comment and explain failed PRT and I'll merge your PR

**Pull request merged**